Packs/SecureWorks/Integrations/TaegisXDR/TaegisXDR.yml

```diff
@@ -66,9 +66,17 @@ script:
   - name: taegis-fetch-alerts
     arguments:
     - name: ids
-      required: true
```

Comment:
Removing this will cause the command to run even when "ids" were not provided. Are you sure this is your intention?

Reply:
Yup, this shouldn't be a problem. The original code used our old Alerts api which did not have a good way of defining the criteria of what alerts would be pulled back, which is why the command originally required the IDs. The new API has better support for searching by specific criteria so the IDs should no longer be required.

With the modified command. we're looking to allow the user to either:

1. Pull back specific alerts based on a list of IDs
2. Pull back all alerts based on criteria. The default criteria has a limit of 10 alerts that are open and with high severity

We have a max number of results that can be returned by the API so I don't believe this should be a problem.